src/uwtools/tests/test_schemas.py

```diff
@@ -12,6 +12,99 @@
 # Fixtures
 
 
+CDEPS_CONFIG = {
```

Comment:
This test config is used both for these schema tests, and for the driver tests in `test_cdeps.py`, above. The values used pass schema validation, but are mostly not very informative -- they do not represent realistic values, especially the ones that just say `"string"`. We could certainly update any of these with something more familiar, but I would need guidance on appropriate values.

Reply:
@maddenp-noaa Some of them are list such as datamode can be only certain values since CDEPS supports set of data modes for each data model. It is hard to put constraint for others such as model_maskfile and model_meshfile. These are just need to point a ESMF mesh file that represent the data file (or domain). I am not sure what is the best approach in here.

Reply:
We can move forward with what's here and of course adjust it later if we develop better ideas.